### PR TITLE
backoffice: add validation errors in response

### PR DESCRIPTION
**Pull request by PascalEgn**
ref: https://github.com/cern-sis/issues-inspire/issues/585
ref: https://github.com/cern-sis/issues-inspire/issues/578

### CODE REVIEW

**Comment by DonHaul** [comment]
Looks good but I think it would be better if we add this logic to the workflow serializer.

File: backoffice/backoffice/workflows/api/utils.py [hunk 1]
```diff
@@ -1,11 +1,22 @@
-from backoffice.workflows.api.serializers import DecisionSerializer
+from backoffice.workflows.api import serializers
 
 
 def add_decision(workflow_id, user, action):
-    serializer_class = DecisionSerializer
```
Comment (DonHaul) [inline backoffice/backoffice/workflows/api/utils.py:5]: why this change

File: backoffice/backoffice/workflows/tests/cassettes/TestAuthorWorkflowViewSet.test_validate_not_valid_record.yaml [hunk 2]
```diff
@@ -1,4 +1,28 @@
 interactions:
+- request:
```
Comment (DonHaul) [inline backoffice/backoffice/workflows/tests/cassettes/TestAuthorWorkflowViewSet.test_validate_not_valid_record.yaml:2]: es responses should not be recorded. in the `conftest.py` could you rename in the `ignore_hosts` `opensearch` to  `es` or alternatively find all instances of `es` and rename the to `opensearch`, and then rerun the tests? (my bad on this one, forgot to change this naming during the monorepoing)

File: record-editor/src/app/backoffice-editor/backoffice-editor.component.ts [hunk 3]
```diff
@@ -86,8 +86,16 @@ export class BackofficeEditorComponent
...
   }
 
+  private injectValidationErrorsIntoExtraData() {
+    this.workflowObject._extra_data.validation_errors = this.workflowObject.validation_errors;
+  }
+
   get workflowExtraData() {
-    return this.workflowObject._extra_data || {};
+    this.workflowObject._extra_data = this.workflowObject._extra_data || {};
+    if (this.workflowObject.validation_errors) {
```
Comment (DonHaul) [inline record-editor/src/app/backoffice-editor/backoffice-editor.component.ts:95]: why do we now require this?

File: record-editor/src/app/shared/interfaces/workflow-object.ts [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { JsonPatch } from './json-patch';
 
 export interface WorkflowObject<T = object> {
   id: number;
+  validation_errors?: Array<ValidationError>;
   _extra_data?: {
     validation_errors?: Array<ValidationError>;
```
Comment (DonHaul) [inline record-editor/src/app/shared/interfaces/workflow-object.ts:8]: should this field be removed?

File: backoffice/backoffice/workflows/api/utils.py (hunk 1 above)
Comment (PascalEgn) [reply backoffice/backoffice/workflows/api/utils.py:5]: Its a circular import otherwise

File: record-editor/src/app/backoffice-editor/backoffice-editor.component.ts (hunk 3 above)
Comment (PascalEgn) [reply record-editor/src/app/backoffice-editor/backoffice-editor.component.ts:95]: As described in https://github.com/cern-sis/issues-inspire/issues/585, the plan is to add the `validation_errors` field into the backend response (because we dont want to have this `extra_data` field anymore). But as we dont want to change much in the record editor (as it need to work with next and backoffice records), we inject the `validation_errors` field into the `extra_data` field

File: record-editor/src/app/shared/interfaces/workflow-object.ts (hunk 4 above)
Comment (PascalEgn) [reply record-editor/src/app/shared/interfaces/workflow-object.ts:8]: field was added because we send the `validation_errors` directly instead of a field in `extra_data` in case of backoffice responses. But we cant remove it from `extra_data` either because we send in like this in case of next responses

File: backoffice/.envs/local/.django [hunk 5]
```diff
@@ -20,7 +20,7 @@ OPENSEARCH_HOST=es:9200
 OPENSEARCH_INDEX_PREFIX=backoffice-backend-local
 
 # Airflow
-AIRFLOW_BASE_URL=http://airflow-webserver:8070
+AIRFLOW_BASE_URL=http://airflow-webserver:8080
```
Comment (drjova) [inline backoffice/.envs/local/.django:23]: wouldn't this be a conflict with ui? Hence we would not be able to run both inspirehep UI and airflow at the same time

Comment (PascalEgn) [reply backoffice/.envs/local/.django:23]: It doesnt because it's the container port (so internally) and not the exposed one. So from the outside you can reach ui under port 8080 and airflow under 8070

File: backoffice/backoffice/workflows/api/utils.py [hunk 6]
```diff
@@ -1,11 +1,22 @@
-from backoffice.workflows.api.serializers import DecisionSerializer
+from backoffice.workflows.api import serializers
```
Comment (drjova) [inline backoffice/backoffice/workflows/api/utils.py:1]: since it's one, I wouldn't import the whole module

**Review by drjova — APPROVED** [review]
Only a small change, otherwise LGTM

File: backoffice/backoffice/workflows/api/utils.py (hunk 6 above)
Comment (PascalEgn) [reply backoffice/backoffice/workflows/api/utils.py:1]: If I only import one, Its a circular import otherwise, therefore the change
